src/resources/parser/glb-parser.js

```diff
@@ -1296,6 +1266,33 @@ const createMaterial = function (gltfMaterial, textures, flipV) {
         material.diffuseVertexColor = false;
     }
 
+    // handle KHR_materials_specular absorption
+    if (gltfMaterial.hasOwnProperty('extensions') &&
+        gltfMaterial.extensions.hasOwnProperty('KHR_materials_specular')) {
```

Comment:
I think we should split these extension handling code out to separate functions.